templates/_common.tpl

```diff
@@ -86,6 +86,16 @@ volumes:
     secret:
       secretName: {{ .Values.postgres.ssl.rootCertSecret.name | quote }}
 {{- end }}
+{{- if ne .Values.coderd.clientSsl.certSecret.name "" }}
```

Comment:
why ssl instead of tls?

Reply:
we should probably use tls here. the postgres certs all reference ssl because that's how they're named in the postgres documentation and settings.

Reply:
I did it to match PG but if that's specific to their docs, I'll make it TLS.